README.rst

```diff
@@ -56,7 +56,69 @@ Features
 Installation
 ============
 
-TBD (Helm)
+Requirements
+------------
+
+In order to install ETOS you need to meet the following requirements.
```

Comment:
Add the missing comma after the introductory phrase "In order to install ETOS,"

Reply:
Fixed.